js/samples/04.ai-apps/h.datasource-azureOpenAI/src/app.ts

```diff
@@ -84,15 +80,10 @@ app.error(async (context: TurnContext, err: any) => {
     await context.sendActivity('To continue to run this bot, please fix the bot source code.');
 });
 
-app.activity(ActivityTypes.Message, async (context: TurnContext, state: ApplicationTurnState) => {
-    const response = await planner.completePrompt(context, state, 'chat');
-
-    if (response.status == 'error') {
-        // If completion response was unsuccessful `response.error` will have the error object.
-        throw response.error;
+app.feedbackLoop(async (context, state, feedbackLoopData) => {
+    if (feedbackLoopData.actionValue.reaction === 'like') {
+        console.log('👍');
```

Comment:
do we still want these console logs here?

Reply:
This is just an example of setting up the handler, so the console logs are intentional - according to Yoga, how feedback is handled/stored is decided by the developer.